core/src/components/datetime/datetime.tsx

```diff
@@ -1362,6 +1366,18 @@ export class Datetime implements ComponentInterface {
         translucent
         overlayIndex={1}
         arrow={false}
+        onWillPresent={ev => {
+          /**
+           * Intersection Observers do not consistently fire between Blink and Webkit
+           * when toggling the visibility of the popover and trying to scroll the picker
+           * column to the correct time value.
+           *
+           * This will correctly scroll the element position to the correct time value,
+           * before the popover is fully presented.
+           */
+          const cols = (ev.target! as HTMLElement).querySelectorAll('ion-picker-column-internal');
+          cols.forEach(col => col.scrollActiveItemIntoView());
```

Comment:
I believe this should replace the IO behavior, but believe further discovery is needed to fully vet that out (created a spike for that, referencing this PR). 

The picker column internal component is only used in context of the datetime and only shown within a popover, so the popover should be able to dictate all visibility-related functions with lifecycle events, instead of relying on an inner IO. I'm just unsure if picker-column-internal is intended to move to the public API for available components.

Reply:
Ultimately I'd like for picker-column-internal to replace the public picker-column since there are many issues with the current implementation. However, I think that is a ways off, so it should be fine to roll with this approach for now.

The only thing I'd ask is we add a TODO and make tech debt to investigate a way of having this be handled automatically. If we make this component public, I imagine users will run into this same issue.